lab_1/main.py

```diff
@@ -49,9 +83,28 @@ def get_top_n_words(freq_dict: dict, top_n: int) -> list:
     top_n = 1
     --> ['happy']
     """
-    pass
+    freq_list = list(freq_dict.items())
+    freq_list.sort(key=lambda x: -x[1])
+    new_freq_dict = dict(freq_list)
+    new_freq_list = list(new_freq_dict.keys())
+    index = 0
+    top_n_words_list = []
```

Comment:
It is usually considered a bad style to include variable type in the variable name -> please, rename to smth like `top_n_words`